index.html

```diff
@@ -222,7 +222,7 @@ <h1 id="head">Code Magic</h1>
           </div>
 
           <label class="radius" for="gradient-border-radius">
-            <input type="checkbox" id="gradient-border-radius" name="check" />
+            <input type="checkbox" id="gradient-border-radius" name="check" onclick="changeDisplay()"/>
```

Comment:
It's considered bad practice to call functions from within your HTML. Take a look at the picture below for clarification.
![Screenshot (72)](https://user-images.githubusercontent.com/72257135/193916949-d27bc3b4-3d2d-486e-98ca-596ac17fea2d.png)